src/cpr_data_access/yql_builder.py

```diff
@@ -123,11 +121,11 @@ def build_where_clause(self) -> str:
         filters.append(self.build_search_term())
         filters.append(self.build_family_filter())
         filters.append(self.build_document_filter())
-        if kf := self.params.keyword_filters:
-            filters.append(self._inclusive_keyword_filters(kf, "family_geography"))
-            filters.append(self._inclusive_keyword_filters(kf, "family_category"))
-            filters.append(self._inclusive_keyword_filters(kf, "document_languages"))
-            filters.append(self._inclusive_keyword_filters(kf, "family_source"))
+        if kf := self.params.filters:
```

Comment:
`kf` might be a confusing variable name now that we're using 
`filters` instead of `keyword_filters`

Reply:
Great spot, I'll update that